graylog2-web-interface/src/components/indices/IndexSetsComponent.jsx

```diff
@@ -60,18 +60,26 @@ const IndexSetsComponent = React.createClass({
   },
 
   _formatIndexSet(indexSet) {
+    const menuItems = [];
+
+    if (indexSet.writable) {
+      menuItems.push(<MenuItem onSelect={this._onSetDefault(indexSet)} disabled={!indexSet.writable}>Set as default</MenuItem>);
```

Comment:
With this change the menu item would never be disabled, this line of code would not be reached for non-writable index sets.

By the way, I like better the idea of having some `title` saying why the item is disabled instead of just removing it from the menu. It would make it more clear imho.